conda_build/post.py

```diff
@@ -485,31 +485,40 @@ def mk_relative_linux(f, prefix, rpaths=('lib',), method='LIEF'):
     elf = os.path.join(prefix, f)
     origin = os.path.dirname(elf)
 
+    existing, _, _ = get_rpaths_raw(elf)
     patchelf = external.find_executable('patchelf', prefix)
-    try:
-        existing = check_output([patchelf, '--print-rpath', elf]).decode('utf-8').splitlines()[0]
-    except CalledProcessError:
-        print('patchelf: --print-rpath failed for %s\n' % (elf))
-        return
+    if not patchelf:
+        print("ERROR :: You should install patchelf, will proceed with LIEF for {} (was {})".format(elf, method))
```

Comment:
Is it actually an error?  If not, we should change ERROR to WARNING
```suggestion
        print("WARNING :: You should install patchelf, will proceed with LIEF for {} (was {})".format(elf, method))
```

Reply:
Maybe if you explicitly ask for patchelf (rather than leaving it as default/None), which you'd do because LIEF is known to be failing, then I think we probably should error out?

Reply:
makes sense.  My main point is: if you show a message and continue, that's a warning.  If you show a warning and stop, that's an error.